Sources/IBLinterKit/Config/Config.swift

```diff
@@ -76,12 +76,12 @@ public struct Config: Codable {
         disableWhileBuildingForIB = try container.decodeIfPresent(Bool.self, forKey: .disableWhileBuildingForIB) ?? true
     }
 
-    public static func load(_ url: URL) throws -> Config {
-        return try YAMLDecoder.init().decode(from: String.init(contentsOf: url))
+    public init(_ url: URL) throws {
+        self = try YAMLDecoder().decode(from: String.init(contentsOf: url))
     }
 
-    public static func load(from configPath: URL, fileName: String = fileName) throws -> Config {
+    public init(from configPath: URL, fileName: String = fileName) throws {
```

Comment:
The parameter name of this initializer is a little bit difficult to understand the behavior. (I named it before. My bad. 🙇 but I don't have any idea of good naming)

Reply:
agree
maybe the first one must be `init(url: URL)`
the second one `init(directoryURL: url, fileName: String = fileName)...`

Reply:
I prefer the former one.